### PR TITLE
Added required redhat-rpm-config

### PR DESCRIPTION
**Pull request by vatsalparekh**

### CODE REVIEW

File: developer_setup.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
   ```bash
   sudo dnf -y group install "C Development Tools and Libraries"  # For unf Gem and noi4r Gem
+  sudo dnf -y install redhat-rpm-config                             # required for version after Fedora 22
```
Comment (bdunne) [inline developer_setup.md]: Why is this required?  I'm on Fedora 27 and don't have this rpm installed.

Comment (vatsalparekh) [reply developer_setup.md]: In my case, Fedora 27 Cloud image didn't had this, and was giving error 
`gcc: error: /usr/lib/rpm/redhat/redhat-hardened-cc1: No such file or directory`
So installing this package would fix that.

Comment (bdunne) [reply developer_setup.md]: What was being compiled when this failed?  I also don't have this file, but don't remember anything failing.

**Comment by martinpovolny** [comment]
@vatsalparekh : closing due to long inactivity. Please, reopen if you are still interested. Thank you!
